lnrpc/routerrpc/router_server.go

```diff
@@ -872,10 +872,17 @@ func (s *Server) trackPayment(subscription routing.ControlTowerSubscriber,
 		stream.Context(), subscription, noInflightUpdates, stream.Send,
 	)
 
+	// If the context is canceled, we don't return an error.
 	if errors.Is(err, context.Canceled) {
-		log.Debugf("Payment stream %v canceled", identifier)
+		log.Infof("Payment stream %v canceled", identifier)
+
+		return nil
```

Comment:
wait why is context canceled not seen as an error anymore?

Reply:
have doubts too - I think `context.Canceled` is the only way that the stream can exit, which means it's not an unexpected erroneous state. For instance, when we shut down `lnd`, this error is always returned, kinda like `EOF`.

Reply:
cool :)